plugins/QnA/class.qna.plugin.php

```diff
@@ -264,10 +264,11 @@ public function discussionController_commentOptions_handler($sender, $args) {
             return;
         }
 
-        if (!Gdn::session()->checkPermission('Vanilla.Discussions.Edit', true, 'Category', $discussion->PermissionCategoryID)) {
-            return;
+        if (!Gdn::session()->checkRankedPermission('Garden.Curation.Manage')) {
```

Comment:
Can you just do this as an `||` rather than a nested `if`?

Reply:
!(A || B) = !A && !B much better than a nested if.